src/org/linphone/LinphoneManager.java

```diff
@@ -167,21 +171,34 @@ public static void removeListener(LinphoneChatMessage.LinphoneChatMessageListene
 	}
 
 	protected LinphoneManager(final Context c) {
+
+		String user = AccountHelper.getAccount(c);
 		sExited = false;
 		mServiceContext = c;
 		basePath = c.getFilesDir().getAbsolutePath();
-		mLPConfigXsd = basePath + "/lpconfig.xsd";
+//
+//		if(LinphoneManager.user!=null)
+//		{
+//			basePath += "/" + LinphoneManager.user ;
+//		}
+//		else
+//		{
+//
+//		}
+
+		mLPConfigXsd = basePath + "/lpconfig"+user+".xsd";
 		mLinphoneFactoryConfigFile = basePath + "/linphonerc";
-		mLinphoneConfigFile = basePath + "/.linphonerc";
+		mLinphoneConfigFile = basePath + "/.linphonerc" + user;
 		mLinphoneRootCaFile = basePath + "/rootca.pem";
 		mRingSoundFile = basePath + "/oldphone_mono.wav";
 		mRingbackSoundFile = basePath + "/ringback.wav";
 		mPauseSoundFile = basePath + "/toy_mono.wav";
-		mChatDatabaseFile = basePath + "/linphone-history.db";
-		mFriendsDatabaseFile = basePath + "/linphone-friends.db";
+		mChatDatabaseFile = basePath + "/linphone-history"+user+".db";
+		mFriendsDatabaseFile = basePath + "/linphone-friends"+user+".db";
```

Comment:
Suggest adding a "-" in there to have a delimiter before the user name, for the sake of any humans that may see it.

"/linphone-historyianblenke.db" seems a bit like a run-on, maybe it would be better as "/linphone-history-ianblenke.db"